appveyor.yml

```diff
@@ -1,20 +1,33 @@
 environment:
   SAXON_CP: '%TEMP%\xspec\saxon\saxon9he.jar'
   matrix:
-    # latest Saxon 9.7 version
+    # latest Saxon 9.7 version and XML Calabash
     - SAXON_VERSION: 9.7.0-15
+      XMLCALABASH_VERSION: 1.1.15-97

```

Comment:
At least, it would be preferable to move the variable `XMLCALABASH_VERSION` to `matrix`. That would make it instant to add a different XML Calabash version to the test set, if we need to do so.